ptx/sec_newton.ptx

```diff
@@ -483,7 +525,18 @@
     <figure xml:id="fig_newt2a">
       <caption>A graph of <m>f(x) = x^3-x^2-1</m>, showing why an initial approximation of <m>x_0=0</m> with Newton's Method fails</caption>
       <image xml:id="img_newt2a" width="47%">
-        <description></description>
+        <shortdescription> 
+          A cubic graph features an "S" shaped curve  with two turning points.
+        </shortdescription>
+        <description>
+            <p>
+            The Y-axis represents function values (0.5, 0, -0.5, -1,-1.5), while the X-axis represents values of (0,0.5,1,1.5)
+            The graph intersects the y-axis at -1 amd 1.5 on x-axis and forms a an "S" shaped curve.
```

Comment:
Replace `"S"` with `<q>S</q>`

Typo: 'amd'

Math tags needed around numbers and variable names.